watchtower/blob/type.go

```diff
@@ -50,6 +57,11 @@ const (
 	// controlled by the user, and does not give the tower a reward.
 	TypeAltruistCommit = Type(FlagCommitOutputs)
 
+	// TypeAltruistAnchorCommit sweeps only commitment outputs from an
+	// anchor commitment to a sweep address controlled by the user, and does
+	// not give the tower a reward.
+	TypeAltruistAnchorCommit = Type(FlagCommitOutputs | FlagAnchorChannel)
```

Comment:
Why no `TypeRewardAnchorCommit`? Also why didn't we need to change these once we added support for static remote key?